invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentsSearch/DocumentsSearch.js

```diff
@@ -15,13 +11,16 @@ import {
 import {
   Error as IlsError,
   SearchBar as DocumentsSearchBar,
-  ResultsSort,
+  SearchControls,
 } from '../../../common/components';
 import { document as documentApi } from '../../../common/api/documents/document';
 import { getSearchConfig } from '../../../common/config';
-import { ClearButton, NewButton } from '../components/buttons';
+import { NewButton } from '../components/buttons';
```

Comment:
If it is not in the common components, we don't need to introduce yet another level `buttons` for local components. If its in the common ones, it can be imported with the rest of other `common/components`